### PR TITLE
KAA-1607: [Reviewed] Using Kaa C SDK, Intel Edison. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 1]
```diff
@@ -10,49 +10,64 @@ sort_idx: 60
...
+   ```
+   
+2. Install the toolchain.
+
+   ```bash
+   cd i686
+   ./install_script.sh
+   ```
+   
+    While running the installation script, you may get an error message: `find: invalid mode '+111'`
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: Missed dot in the end of the sentence.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 2]
```diff
@@ -10,49 +10,64 @@ sort_idx: 60
...
-But remember, you must specify correct compiler when compiling your Kaa application for Intel Edison:
-
-        cmake -DKAA_MAX_LOG_LEVEL=3 -DCMAKE_TOOLCHAIN_FILE=PATH_TO_KAA_SDK/toolchains/edison.cmake -DEDISON_SDK_ROOT=PATH_TO_EDISON_SDK -DBUILD_TESTING=OFF ..
-        make
-
->**NOTE:** `PATH_TO_KAA_SDK` should be replaced with the path to Kaa C SDK relative to the `build` directory
-and `PATH_TO_EDISON_SDK` should be replaced with the absolute path to the Edison SDK installation directory (see above).
-
-For more details on how to build, upload and run your application on Edison board, you may refer to official [user guide](https://software.intel.com/en-us/intel-edison-board-user-guide).
+This guide explains how to cross-compile [Kaa C SDK]({{root_url}}Glossary/#kaa-sdk-type) for [Raspberry Pi](https://www.raspberrypi.org/).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: for Intel Edison

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 3]
```diff
@@ -10,49 +10,64 @@ sort_idx: 60
...
+Alternatively, you can build the Kaa C SDK directly on the Raspberry Pi board.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: Edison board

**Review by Polinaa — APPROVED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: Fixed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: Fixed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: Fixed.

**Review by vadimol — APPROVED** [review]

**Pull request merged**